app/models/manageiq/providers/microsoft/infra_manager.rb

```diff
@@ -101,9 +101,7 @@ def vm_resume(vm, _options = {})
   end
 
   def vm_destroy(vm, _options = {})
-    if vm.power_state == "on"
-      vm_stop(vm)
-    end
+    vm_stop(vm)
```

Comment:
@gmcculloug - Yes, without the `-RunAsynchronously` parameter it is synchronous.
See **fix** section of my PR description for the answer to your second question :)